### PR TITLE
Rk

### PR DESCRIPTION
**Pull request by canyon289**
@OriolAbril I edited your bio to be more direct since I don't think you're trying to do the thing you do, but rather just doing them directly ;) 

Leaving this up for your review to see if you agree

### CODE REVIEW

File: content/team/oriol-abril-pla/contents.lr [hunk 1]
```diff
@@ -6,7 +6,7 @@ github_url: https://github.com/OriolAbril
 ---
 location: Barcelona, Spain
 ---
-blurb: Trying to develop and apply new methodology for a robust and reproducible statistical analysis workflow.
+blurb: Devloping and applying new methodology for a robust and reproducible statistical analysis workflow.
```
Comment (twiecki) [inline content/team/oriol-abril-pla/contents.lr]:
```suggestion
blurb: Developing and applying new methodology for a robust and reproducible statistical analysis workflow.
```

File: content/team/ravin-kumar/contents.lr [hunk 2]
```diff
@@ -5,3 +5,14 @@ picture: https://avatars0.githubusercontent.com/u/7213793?s=400&u=a3ce97892d6f56
 github_url: https://github.com/canyon289
 ---
 location: Los Angeles, USA
+---
+blurb: Trying to develop and apply new methodology for a robust and reproducible statistical analysis workflow.
```
Comment (twiecki) [inline content/team/ravin-kumar/contents.lr]: Isn't this the same blurb that you changed for Oriol? :D

File: content/team/ravin-kumar/contents.lr [hunk 3]
```diff
@@ -5,3 +5,14 @@ picture: https://avatars0.githubusercontent.com/u/7213793?s=400&u=a3ce97892d6f56
...
+blurb: Using Bayesian methods and uncertainty to develop datat workflows to aid decisions in strategic decision making in organizations
```
Comment (twiecki) [inline content/team/ravin-kumar/contents.lr]:
```suggestion
blurb: Using Bayesian methods and uncertainty to develop data workflows to aid decisions in strategic decision making in organizations
```

File: content/team/ravin-kumar/contents.lr (hunk 2 above)
Comment (OriolAbril) [reply content/team/ravin-kumar/contents.lr]: This looks like too aggressive copy pasting, one of the blurbs should be deleted for the website to render correctly

Comment (OriolAbril) [inline content/team/ravin-kumar/contents.lr]:
```suggestion
```

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**